frontend/src/components/pages/WaitlistExperience/index.tsx

```diff
@@ -1,35 +1,130 @@
-import React, { useState } from "react";
-import WaitlistConfirmationPage from "./ConfirmationPage";
+import { Box, Center, Flex, Spinner } from "@chakra-ui/react";
+import React, { useRef, useState } from "react";
+import { RegistrationCamper } from "../../../types/CamperTypes";
+import { CampResponse, CampSession } from "../../../types/CampsTypes";
+import EmptyNavBar from "../../common/EmptyNavBar";
+
+import RegistrationFooter from "../RegistrantExperience/RegistrationSteps/RegistrationFooter";
+import WaitlistConfirmationPage from "./ConfirmationPage/ConfirmationPage";
 import WaitlistPersonalInfoPage from "./PersonalInfo";
-import WaitlistFooter from "./WaitlistFooter";
+import checkPersonalInfoFilled from "./PersonalInfo/checkPersonalInfoFilled";
+import WaitListExperienceSteps from "./WaitListExperienceSteps";
+
+interface WaitlistExperiencePageProps {
+  camp: CampResponse;
+  selectedCampSessions: CampSession[];
+  onClickBack: () => void;
+}
+
+const waitListNextStepTxtMap = new Map();
+waitListNextStepTxtMap.set(WaitListExperienceSteps.PersonalInfoPage, "Submit");
+
+const WaitlistExperiencePage = ({
+  camp,
+  selectedCampSessions,
+  onClickBack,
+}: WaitlistExperiencePageProps): React.ReactElement => {
+  const nextBtnRef = useRef<HTMLButtonElement>(null);
+  const [currentStep, setCurrentStep] = useState<WaitListExperienceSteps>(
+    WaitListExperienceSteps.PersonalInfoPage,
+  );
+  const [campers, setCampers] = useState<RegistrationCamper[]>([
```

Comment:
There is currently a `CreateWaitlistedCamperDTO` type defined in `frontend/src/types/CamperTypes.ts`. Can we use this type instead? 

When we make the backend call to create the waitlisted campers, the `waitlistCampers` function in the `frontend/src/APIClients/CamperAPIClient.ts` will need the data to be sent as an array of `CreateWaitlistedCamperDTO`